package.json

```diff
@@ -18,21 +18,28 @@
     "index.*"
```

Comment:
@unchai one last thing: we need to add `dist/index.js` to here

Reply:
agree ;)

Reply:
@fouad @slorber Sorry for late. I just added it. :)